frontend/src/App.js

```diff
@@ -2,6 +2,7 @@ import './App.css';
 import LandingPage from './components/LandingPage/LandingPage';
 import CompanyInput from './components/CompanyInput/CompanyInput';
 import MyNavbar from "./components/MyNavbar/MyNavbar";
+import StockDetails from './components/StockDetails/StockDetails'
```

Comment:
Change to **StockQuote**. **StockDetails** does not exist in any file or folder; there is only **StockQuote**.

Reply:
Hey Dylan so this PR would have to be merged after cb-25 for it to work. That is where Stockdetails.js is located. I wanted to break up that PR so that it wouldnt be so big.